bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -313,7 +313,9 @@ public void run() {
                 LOG.debug("Acknowledge Ledger: {}, Entry: {}", ledgerId, entryId);
             }
             journalCbQueueSize.dec();
-            journalAddEntryStats.registerSuccessfulEvent(MathUtils.elapsedNanos(enqueueTime), TimeUnit.NANOSECONDS);
+            if (journalAddEntryStats != null) {
```

Comment:
when this can be null?

if this can be null, that is a bug. the whole purpose of having NullStatsLogger is to avoid this null checking.

Reply:
I am setting it to null because forceLedger is not addEntry can add another logger if you prefer

Reply:
introduced new logger (JOURNAL_FORCE_LEDGER)